cdk/README.md

```diff
@@ -36,17 +36,28 @@ $ pip install -r requirements.txt
 
 You must now ensure that you are signed into AWS CLI:
 
-Instructions for configuring AWS CLI: 
-- https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html
+Instructions for configuring AWS CLI:
 
-Sign in instructions for AWS Educate accounts:
-- Navigate to awseducate.com and sign in
-- Go to "My Classrooms" and select go to classroom for the class' account you wish to deploy to
-- Select "Account Details" and "Show" next to AWS CLI
-- Copy and paste the block code that is shown into ~/.aws/credentials
+-   https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html
+
+In order to sign into your AWS account on the CLI, follow one of the 3 options below:
+
+    1. Configure the AWS CLI to use AWS SSO. This will allow you to use the AWS CLI without having
+    to manually navigate to the login page. To do this, follow the instructions in [this guide](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-sso.html#sso-using-profile). Keep in mind that if you go this route, in order
+    to get emporary credentials you will need ot run `aws sso login --profile your-profile-name` before each command.
```

Comment:
Here's a suggestion to fix some minor typos:
```suggestion
    to get temporary credentials you will need to run `aws sso login --profile <your-profile-name>` before each command.
```

Also, are you sure you'll need to run `aws sso login --profile <your-profile-name>` before each command? I would expect that you should only have to run it once at the start of each session.

Reply:
I've made a commit to fix this typo, but can you take a look at my previous question?

> are you sure you'll need to run aws sso login --profile <your-profile-name> before each command? I would expect that you should only have to run it once at the start of each session.

Reply:
So in my case I needed to run it before each command. Lemme mess with my `~/.aws/credentials` I think it might be because I also have a personal AWS Educate account on my computer, so that's how it handles the switching.

Reply:
Yep, that's a me thing. Do you think it'd be worth it to include instructions for if you are adding it as a second account (like if they already has an AWS Educate account)? I can modify it to include those directions then. @ophilli

Reply:
It might be nice to have those instructions, but I wouldn't worry about it. No need to block this PR for that.